tests/docker/languages/Dockerfile.testing.rust.big_endian.1_30_1

```
@@ -0,0 +1,15 @@
+FROM rust:1.30.1-slim-stretch as base
+
+WORKDIR /setup
+RUN apt -qq update >/dev/null
+RUN apt -qq install -y curl gcc-mips-linux-gnu
+RUN curl https://sh.rustup.rs > rustup.sh
+RUN cat rustup.sh | sh -s -- -y --default-toolchain none
+RUN rustup target add mips-unknown-linux-gnu
+
+
+WORKDIR /code
+ADD . .
+WORKDIR /code/tests/rust_usage_test
+ENV CARGO_TARGET_MIPS_UNKNOWN_LINUX_GNU_LINKER=mips-linux-gnu-gcc
+RUN cargo build --target mips-unknown-linux-gnu
```

Need to tell cargo how to run the MIPS binaries.

```suggestion
ENV CARGO_TARGET_MIPS_UNKNOWN_LINUX_GNU_RUNNER="qemu-mips -L /usr/mips-linux-gnu"
RUN cargo test --target mips-unknown-linux-gnu
```